src/test/java/core/basesyntax/FruitStorageTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import core.basesyntax.exception.NotEnoughFruitException;
+import core.basesyntax.model.Fruit;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+import static org.junit.Assert.*;
```

Comment:
import with * considered to be a bad practice

Reply:
fixed